pkg/proc/proc_unix_test.go

```diff
@@ -3,27 +3,70 @@
 package proc
 
 import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
 	"syscall"
 	"testing"
 	"time"
 
 	protest "github.com/derekparker/delve/pkg/proc/test"
 )
 
+// Checks that p.Pid() is the pid of fixture.
+func checkPid(t *testing.T, p *Process, fixture protest.Fixture) error {
+	if p.Pid() <= 0 {
+		return fmt.Errorf("zero or negative: %d", p.Pid())
```

Comment:
We're passing in `*testing.T` here, let's just fail the test instead of returning an error.